src/metabase/util/schema.clj

```diff
@@ -350,6 +351,27 @@
                            s/Keyword                 s/Any}
     (deferred-tru "parameter_mapping must be a map with :parameter_id and :target keys")))
 
+(def ^:private TemplateTagSchema
+  ;; For the exact schema, check [[metabase.mbql.schema/TemplateTagMap]]
```

Comment:
I don't understand this comment. (I'm a newbie here.)

Reply:
the schema defined here is primarily used for APIs, which it's basically is a looser version of the schema we use in MBQL.

The comment is a bit confusing. I'll update it